Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -293,161 +381,11 @@ def gen_xml_file(stitch_dir, cfg_var, btg_profile, spi_quad):
 
     tree = ET.parse(os.path.join(stitch_dir, cfg_var['fitinput'], 'new.xml'))
 
-    #Region Order
```

Comment:
Platform is added gen_xml_file(), but it is not used when generate xml file.
Different SKU will have different xml file. So should we pass SKU/platform info to fit command in line 379?

Reply:
True, Fixed this.